Cargo.toml

```
@@ -1,35 +1,22 @@
-[package]
-name = "roaring"
-version = "0.10.6"
-rust-version = "1.65.0"
-authors = ["Wim Looman <wim@nemo157.com>", "Kerollmops <kero@meilisearch.com>"]
-description = "A better compressed bitset - pure Rust implementation"
+[workspace]
+members = ["roaring", "benchmarks"]
+resolver = "2"
 
-documentation = "https://docs.rs/roaring"
-repository = "https://github.com/RoaringBitmap/roaring-rs"
+[workspace.dependencies]
+roaring = { path = "roaring" }
 
-readme = "README.md"
-keywords = ["roaring", "data-structure", "bitmap"]
-categories = ["data-structures"]
-edition = "2021"
-
-license = "MIT OR Apache-2.0"
-
-[dependencies]
-bytemuck = { version = "1.16.1", optional = true }
-byteorder = { version = "1.5.0", optional = true }
-serde = { version = "1.0.203", optional = true }
-
-[features]
-default = ["std"]
-serde = ["dep:serde", "std"]
-simd = []
-std = ["dep:bytemuck", "dep:byteorder"]
-
-[dev-dependencies]
+bincode = "1.3.3"
+bytemuck = "1.16.1"
+byteorder = "1.5.0"
+criterion = "0.5"
```

```suggestion
criterion = "0.3"
```

It seems that the CI is not happy with the rustic version because you bumped criterion to v0.5 but you need to keep v0.3.

---

Thanks for the review!

Good catch. I elegantly ignored the MSRV declaration. I had to downgrade one more dependency. 
`cargo +1.65.0 build --release && cargo +1.65.0 bench` run successfully now. I did not run the  benchmarks to the end, but they successfully started up. 

I have added the MSRV to the benchmarking crate as well. Is there a specific reason that the roaring-rs crate MSRV [is set to `1.65.0`](https://github.com/RoaringBitmap/roaring-rs/blob/main/Cargo.toml#L4), but the CI runs with `1.66.0`?

---

> Is there a specific reason that the roaring-rs crate MSRV [is set to `1.65.0`](https://github.com/RoaringBitmap/roaring-rs/blob/main/Cargo.toml#L4), but the CI runs with `1.66.0`?

I don't think there is any good reason for that. We should maybe unify them at some point.